client/web/compose/src/mixins/record.js

```diff
@@ -281,8 +281,14 @@ export default {
 
       const { moduleID, namespaceID } = this.module
 
+      let query = ''
+
+      if (!includeAllRecords) {
```

Comment:
Wouldn't it make sense to always evaluate the query? If records is empty then you know you're processing them all. So in essence, `query = records.map(...` in all cases